### PR TITLE
k8s/slim: Clarify instructions for updating slim files

### PR DESCRIPTION
**Pull request by christarazi**
Signed-off-by: Chris Tarazi <chris@isovalent.com>

### CODE REVIEW

File: pkg/k8s/slim/README.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ This package is intended to be a slim copy of the structures used in k8s watcher
...
 `vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go`.
 
+Use the following script to update the slim files. Update the `tag` variable as
+necessary. Keep in mind that updating the slim files is not as simple as
+accepting the changes that the script generates. Only the changes relevant to
+the structures used by Cilium are necessary to keep.
+
+Note: if updating from RC versions, first run the script as-in and commit in a
+temporary commit all of the changes generated by the script. Then, update the
+`tag` and re-run the script. The generated diff is what needs to be committed.
```
Comment (aanm) [inline pkg/k8s/slim/README.md]: It depends on the diff. If the diff is for structures that aren't part of the slim structures then they won't matter to us

Comment (christarazi) [reply pkg/k8s/slim/README.md]: Fixed.

**Review by joamaki — APPROVED** [review]

**Pull request merged**